astropy/coordinates/tests/test_intermediate_transformations.py

```diff
@@ -410,3 +410,30 @@ def test_icrs_altaz_moonish(testframe):
 
     #now check that the distance change is similar to earth radius
     assert 1000*u.km < np.abs(moonaa.distance - MOONDIST).to(u.au) < 7000*u.km
+
+
+@remote_data
+def test_gcrs_self_transform_closeby():
+    """
+    Tests GCRS self transform for objects which are nearby and thus
+    have reasonable parallax.
+
+    Create moon positions using JPL DE432s ephemeris (small download).
+
+    The two lunar positions (one geocentric, one at a defined location)
+    are created via a transformation from ICRS to two different GCRS frames.
+
+    We test that the GCRS-GCRS self transform can correctly map one GCRS
+    frame onto the other.
+    """
+    t = Time("2014-12-25T07:00")
+    moon_geocentric = get_moon(t, ephemeris='de432s')
```

Comment:
actually, looking at this, I don't see why this needs to use the ephemeris at all.  Why not instead just use something like the "moonish" test above?  (i.e., just put something at x=y=z=``MOONDIST``/sqrt(3) - `MOONDIST` is already defined in the test just above)  That way you can dispense with `remote_data`.